dashboard/app/models/course.rb

```diff
@@ -54,7 +54,8 @@ def self.load_from_path(path)
     hash = JSON.parse(serialization)
     course = Course.find_or_create_by!(name: hash['name'])
     course.update_scripts(hash['script_names'], hash['alternate_scripts'])
-    course.update!(teacher_resources: hash.try(:[], 'properties').try(:[], 'teacher_resources'))
+    course.properties = hash['properties']
+    course.save!
```

Comment:
I'd originally tried replacing this with `course.update!(properties: hash.try(:[], 'properties'))`, but `update!` does a merge, and so if you try to update from a non-empty property hash to an empty property hash, you end up with the existing properties, i.e.
```
course.properties = { foo: 'bar' }
course.update!(properties: {})
// course.properties is still {foo: 'bar'}